### ISSUE TITLE
Custom data types cannot be expanded in record mode

### ISSUE THREAD

**Issue by uslss** [dbeaver/dbeaver#17016]
#### System information: 
- Operating system (distribution) and version win10 
- DBeaver version 22.1.2

#### Connection specification:
- Database name and version Postgres

#### Describe the problem you're observing:
Custom data types cannot be expanded in record mode
#### Steps to reproduce, if exist:
![image](https://user-images.githubusercontent.com/31996417/178000019-7166a4f6-09a7-4272-8158-45cbc92fe2f8.png)

**Comment by ShadelessFox** [comment]
Fixed in #17031

**Comment by uslss** [comment]
NPE appears when trying to expand
```
eclipse.buildId=unknown
java.version=11.0.12
java.vendor=Eclipse Foundation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en
Command-line arguments:  -os win32 -ws win32 -arch x86_64

org.jkiss.dbeaver.model
Error
Wed Jul 20 10:38:56 MSK 2022
NullPointerException

java.lang.NullPointerException
	at org.jkiss.dbeaver.model.DBUtils.getAttributeValue(DBUtils.java:768)
	at org.jkiss.dbeaver.ui.controls.resultset.ResultSetModel.getCellValue(ResultSetModel.java:382)
	at org.jkiss.dbeaver.ui.controls.resultset.spreadsheet.SpreadsheetPresentation$ContentProvider.getCollectionSize(SpreadsheetPresentation.java:1894)
	at org.jkiss.dbeaver.ui.controls.lightgrid.LightGrid.toggleRowExpand(LightGrid.java:3396)
	at org.jkiss.dbeaver.ui.controls.lightgrid.LightGrid.onMouseDown(LightGrid.java:3239)
	at org.jkiss.dbeaver.ui.controls.lightgrid.LightGrid$1.mouseDown(LightGrid.java:3034)
	at org.eclipse.swt.widgets.TypedListener.handleEvent(TypedListener.java:196)
	at org.eclipse.swt.widgets.EventTable.sendEvent(EventTable.java:89)
	at org.eclipse.swt.widgets.Display.sendEvent(Display.java:4251)
	at org.eclipse.swt.widgets.Widget.sendEvent(Widget.java:1066)
	at org.eclipse.swt.widgets.Display.runDeferredEvents(Display.java:4068)
	at org.eclipse.swt.widgets.Display.readAndDispatch(Display.java:3645)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine$5.run(PartRenderingEngine.java:1155)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.e4.ui.internal.workbench.swt.PartRenderingEngine.run(PartRenderingEngine.java:1046)
	at org.eclipse.e4.ui.internal.workbench.E4Workbench.createAndRunUI(E4Workbench.java:155)
	at org.eclipse.ui.internal.Workbench.lambda$3(Workbench.java:644)
	at org.eclipse.core.databinding.observable.Realm.runWithDefault(Realm.java:338)
	at org.eclipse.ui.internal.Workbench.createAndRunWorkbench(Workbench.java:551)
	at org.eclipse.ui.PlatformUI.createAndRunWorkbench(PlatformUI.java:156)
	at org.jkiss.dbeaver.ui.app.standalone.DBeaverApplication.start(DBeaverApplication.java:281)
	at org.eclipse.equinox.internal.app.EclipseAppHandle.run(EclipseAppHandle.java:203)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.runApplication(EclipseAppLauncher.java:136)
	at org.eclipse.core.runtime.internal.adaptor.EclipseAppLauncher.start(EclipseAppLauncher.java:104)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:402)
	at org.eclipse.core.runtime.adaptor.EclipseStarter.run(EclipseStarter.java:255)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.base/java.lang.reflect.Method.invoke(Unknown Source)
	at org.eclipse.equinox.launcher.Main.invokeFramework(Main.java:659)
	at org.eclipse.equinox.launcher.Main.basicRun(Main.java:596)
	at org.eclipse.equinox.launcher.Main.run(Main.java:1467)
```
